app/views/play/level/modal/CourseVictoryComponent.vue

```diff
@@ -196,9 +202,16 @@
       nextLevelVideo: ->
         if me.isStudent() and @course._id == utils.courseIDs.INTRODUCTION_TO_COMPUTER_SCIENCE and !me.showHeroAndInventoryModalsToStudents()
           return utils.videoLevels[@nextLevel.original]
+      inExam: ->
+        exam = storage.load('exam', true)
```

Comment:
can we move the whole storage logic to user-utils, there seems to be a lot of repetition. Also, can we have the key as `exam-${me.id}` instead?

Reply:
ok